Update schedules_and_milestones_guidance from contract manager feedback (#2659)
DFE-Digital/register-early-career-teachers-public: app/views/api/guidance/guidance_for_lead_providers/schedules_and_milestones_guidance.md

---
title: Schedules guidance
sidebar_position: 6
---

## Purpose and scope of this guidance

This guidance explains how schedules operate within the register early career teachers service.

> Schedules and milestone or window dates are contractually determined and owned by the Department for Education (DfE). This guidance should be read alongside the Payment Guidance issued by contract management, which remains the authoritative source for payment dates, liability, and contractual requirements.

Reviewer: Should this be "Schedules and window dates"

Author: No, becausemilestone is still accurate for some open cohorts :(

| Concept | Definition |
| ------- | ---------- |

> | Milestone | Contractual retention periods during which providers must submit relevant declarations evidencing training delivery and participant retention. These are used in the delivery of ECF training programme, in 2024 cohorts and earlier. |

Reviewer: ```suggestion
| Milestone | Contractual retention periods during which providers must submit relevant declarations evidencing training delivery and participant retention. These are used in the delivery of the ECF training programme, in 2024 cohorts and earlier. |
```

> | Milestone dates | The deadline date a valid declaration can be made for a given milestone in order for DfE to be liable to make a payment the following month. Milestone dates are dependent on the participant's schedule. These are used in the delivery of ECF training programme, in 2024 cohorts and earlier. |

Reviewer: ```suggestion
| Milestone dates | The deadline date a valid declaration can be made for a given milestone in order for DfE to be liable to make a payment the following month. Milestone dates are dependent on the participant's schedule. These are used in the delivery of the ECF training programme, in 2024 cohorts and earlier. |
```

> | Milestone period | The period of time between the milestone start date and deadline date. These are used in the delivery of ECF training programme, in 2024 cohorts and earlier. This period is also referred to as a declaration window for ECTP, in the 2025 cohort or later. |

Reviewer: ```suggestion
| Milestone period | The period of time between the milestone start date and deadline date. These are used in the delivery of the ECF training programme, in 2024 cohorts and earlier. This period is also referred to as a declaration window for ECTP, in the 2025 cohort or later. |
```

| Extended schedule | A non-standard training schedule for ECTs who expect to complete their induction over a period greater than 2 years.<br><br>Examples include part-time ECTs, or ECTs whose induction period is extended by their appropriate body.<br><br>Mentors can also be assigned extended schedules if they deferred their training and came back later, or they’re doing the training part-time. However, mentors can't receive extended declarations. |
| Reduced schedule | A non-standard training schedule for ECTs who expect to complete their induction over a period less than 2 years. An example is an ECT with previous experience teaching.<br><br>Mentors should not be placed on a reduced schedule. |
| Replacement schedule | A non-standard training schedule for mentors that are replacing a previous mentor for an ECT that is part way through their training.<br><br>The mentor must be completely new to provider-led mentor training. |

> | Standard schedule | The default training schedule for ECTs completing a standard 2 year induction, or mentors completing a standard training for their cohort, starting in September, January or April. (Check we include mentors do 2 years for 2024 and earlier, 1 year later.) |

Reviewer: Would "or mentors completing standard training for their cohort" be better?

Also, regarding the note at the end, do we need to include anything extra? Either way it's going to be standard training for that cohort right?

Hope that makes sense?

Author: fixed I think

> | Standard schedule | The default training schedule for ECTs completing a standard 2 year induction, or mentors completing a standard training for their cohort, starting in September, January or April. (Check we include mentors do 2 years for 2024 and earlier, 1 year later.) |

Author: ```suggestion
| Standard schedule | The default training schedule for ECTs completing a standard 2 year induction, or mentors completing standard training for their cohort, starting in September, January or April. Mentors in pre-2025 cohorts would typically do 2 years of training, whilst after this they would do 1 year of training. |
```

If a school registers a replacement mentor for an ECT, we'll assign a replacement mentor schedule based on when they're first registered as the ECT's mentor (`replacement-september`, `replacement-january` or `replacement-april`).

> It will be up to providers to check these automatically assigned replacement schedules. If a newly assigned replacement mentor later supports a new ECT doing provider-led training as their first mentor before the end of their first term of training, and before the mentor's first started declaration is due, providers must update the mentor's schedule to a standard one.

Reviewer: ```suggestion
It will be up to providers to check these automatically assigned replacement schedules. If a replacement mentor is later assigned as a first mentor to a new ECT doing provider-led training, providers must update the mentor's schedule to a standard one. This must be done before the end of the ECT's first term of training and before the mentor's first started declaration is due.
```

Hope this is accurate. Just trying to break up the length of this sentence.

Author: good shout

## Milestone and payment date guidance

> DfE pays lead providers based on agreed contractual schedules and training delivery criteria. Payments relate to the engagement criteria given to lead provider in their contracts. 

Reviewer: ```suggestion
DfE pays lead providers based on agreed contractual schedules and training delivery criteria. Payments relate to the engagement criteria given to lead providers in their contracts. 
```

> Output payments are based on valid declarations submitted by the declaration deadline for each milestone or declaration window. These dates correspond to each participant's schedule.

Reviewer: Are we keeping 'milestone' here?

Reviewer: Same q for the replacement for line 101

Contract management circulates milestone or declaration window dates and payment guidance to lead providers.

## Example dates for standard September schedule for ECTs

| Milestone | Start date | Declaration deadline | Declaration type | Payment date |
| --------- | ---------- | -------------------- | ---------------- | ------------ |

> | Participant Start | 1 Jun 2025 | 31 Oct 2025 (interim)( <br><br>31 Dec 2025 | started<br><br>started | 30 Nov 2025<br><br>31 Jan 2026 |

Reviewer: ```suggestion
| Participant Start | 1 Jun 2025 | 31 Oct 2025 (interim) <br><br>31 Dec 2025 | started<br><br>started | 30 Nov 2025<br><br>31 Jan 2026 |
```

Just removing a stray bracket.
